packages/hardhat-verify/src/internal/errors.ts

```diff
@@ -9,6 +9,7 @@ import { TASK_VERIFY_VERIFY } from "./task-names";
 export class HardhatVerifyError extends NomicLabsHardhatPluginError {
   constructor(message: string, parent?: Error) {
     super("@nomicfoundation/hardhat-verify", message, parent);
+    Object.setPrototypeOf(this, this.constructor.prototype);
```

Comment:
This is to enable `instanceOf` for code that is catching the error, or is there something else this enables?

Reply:
Yes, without this line, the `instanceof` operator doesn't work correctly for classes that extend `NomicLabsHardhatPluginError`

Reply:
This is because the `NomicLabsHardhatPluginError` constructor is messing with the prototype chain: `Object.setPrototypeOf(this, NomicLabsHardhatPluginError.prototype);`.